package-lock.json

```diff
@@ -7083,13 +6553,6 @@
       "integrity": "sha1-MHXOk7whuPq0PhvE2n6BFe0ee6s=",
       "dev": true
     },
-    "nan": {
-      "version": "2.14.0",
-      "resolved": "https://registry.npmjs.org/nan/-/nan-2.14.0.tgz",
-      "integrity": "sha512-INOFj37C7k3AfaNTtX8RhsTw7qRy7eLET14cROi9+5HAVbbHuIWUHEauBv5qT4Av2tWasiTY1Jw6puUNqRJXQg==",
-      "dev": true,
-      "optional": true
-    },
```

Comment:
Were these intentional?

Reply:
I started working on this branch when the files were not using typescript. Probably happened when I merged upstream with this branch. My work was already stashed by this point, so when I re-applied this to the merged branch. Once I fixed a couple of conflicts and ran the dev server, this change probably occurred. I don't think this is intentional but after checking the develop branch for comparison, I think will merge this branch with upstream dev to fix this.

Reply:
Done.